upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -520,6 +520,34 @@ else
     echo "====> ${state_name} has been completed" | tee -a "${LOG_FILE}"
 fi
 
+state_name="ROLL_SPIRE_POSTGRES"
+state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
+if [[ ${state_recorded} == "0" ]]; then
+    echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
```

Comment:
nit, all the code here is inconsistently indented by 2/4 spaces e.g the echo's don't line up with the general indentation consistently.

Reply:
I had to turn off my autoformat for this file. shfmt changes a ton of stuff. I'll fix formatting.

Reply:
> [CASMPET-6582 14d9c5ad7] CASMPET-6582  Run shfmt
>  1 file changed, 471 insertions(+), 483 deletions(-)

is what happens when my editor autoformats. Way too many changes to put into a PR unless it's the only thing in the PR.